### PR TITLE
Cinder migrate 3.0

### PR DESCRIPTION
**Pull request by nicolasbock**

### CODE REVIEW

File: chef/data_bags/crowbar/migrate/cinder/048_nfs_backend.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+def upgrade(ta, td, a, d)
+  a["volume_defaults"]["nfs"] = ta["volume_defaults"]["nfs"]
```
Comment (Itxaka) [inline chef/data_bags/crowbar/migrate/cinder/048_nfs_backend.rb]: FYI: As discussed on https://github.com/crowbar/crowbar-core/pull/598 migrations that are backported should be modified on master to protect against being run several times and overwriting data :)

Comment (nicolasbock) [inline chef/data_bags/crowbar/migrate/cinder/048_nfs_backend.rb]: Yes, It seemed a bit odd at first, but now I get it :smile:

**Comment by Itxaka** [comment]
LGTM +1

**Comment by jsuchome** [comment]
Why was it reverted earlier?

**Comment by nicolasbock** [comment]
In the original PR, https://github.com/crowbar/crowbar-openstack/pull/497, I had gotten the revision for migration wrong. @toabctl suggested to revert the PR, fix the revision and resubmit.

**Comment by jsuchome** [comment]
Ah, true, this is 3.0 branch. +1

**Pull request merged**